osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/mapping/InstallationMapper.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.domain.smartmetering.application.mapping;
+
+import org.opensmartgridplatform.adapter.domain.smartmetering.application.mapping.customconverters.DeCoupleMbusDeviceByChannelResponseConverter;
+import org.springframework.stereotype.Component;
+
+import ma.glasnost.orika.MapperFactory;
+import ma.glasnost.orika.converter.ConverterFactory;
+import ma.glasnost.orika.impl.ConfigurableMapper;
+
+@Component(value = "installationMapper")
```

Comment:
is there a reason to name this component ?

Reply:
By convention of mapper